services/meme-maker/index.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const generateMemeMakerClient = () => {
+	console.log('Here', process.env);
```

Comment:
nit: let's remove the console. Let's maybe add a logger function like we have for hapi with levels and all.

Reply:
https://www.npmjs.com/package/winston